src/app/lib/controller.js

```diff
@@ -203,9 +206,12 @@ class Controller {
                     this.type = args[0];
                     this.state = { ...args[1] };
                 }
+                if (eventName === 'electron-error') {
+                    window.ipcRenderer.send('log-error', args[0]);
```

Comment:
You need to wrap this in an isElectron check - this will throw an error if users are accessing it from the headless view.  You can send the ipc message if it's electron, or console log/client log if it's web view.